README.rst

```diff
@@ -22,8 +22,8 @@ Faker is heavily inspired by `PHP Faker`_, `Perl Faker`_, and by `Ruby Faker`_.
 Compatibility
 -------------
 
-Starting from version ``4.0.0``, ``Faker`` dropped support for Python 2 and only supports Python
-3.5 and above. If you still need Python 2 compatibility, please install version ``3.0.1`` in the
+Starting from version ``5.0.0``, ``Faker`` dropped support for Python 2 and only supports Python
```

Comment:
The statement still holds true, I don’t think we need to update this paragraph

Reply:
Agreed. Changed back. Thanks.